src/Components/IncidentFilter/IncidentFilter.Component.jsx

```diff
@@ -33,49 +34,53 @@ export default class IncidentFilter extends Component {
 
 
   render() {
+    const styles = {
```

Comment:
We need to move these to the `IncidentFilter.scss` file

Reply:
Material-ui components use inline styling, so if the styles were moved to the IncidentFilter.scss they would not apply to the toggle